src/launcher.ts

```diff
@@ -28,9 +31,34 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
     onPrepare (config?: Options.Testrunner, capabilities?: Capabilities.RemoteCapabilities) {
         if (Array.isArray(capabilities)) {
             capabilities.forEach((capability: Capabilities.DesiredCapabilities | any) => {
-                const wdioServiceVersion = process.env.npm_package_version;
+                const wdioServiceVersion = version;
+                let webdriverIOVersion: any = undefined;
+                const packageFile: any = process.env.npm_package_json;
+                if (packageFile !== undefined) {
```

Comment:
Lets organize this, put the newer code in a new method

Reply:
done